src/Main.java

```diff
@@ -0,0 +1,226 @@
+import Cabinet.Cabinet;
+import Personal.Asistent;
+import Personal.Doctor;
+import Personal.Pacient;
+import Personal.Persoana;
+import Programare.Diagnostic;
+import Programare.Tratament;
+
+
+public class Main implements Service {
```

Comment:
Main shouldn't implement an interface, services implement interfaces. 

Solution: Make the existing "Service" more specific - ideally, create a service interface and a service implementation for each entity that you operate upon.

Then, in Main you just instantiate services and call their methods, without having to implement them.